ML-TOOLS.md

```diff
@@ -0,0 +1,88 @@
+# 🛠️ ML Tools for Computational Text
+
+**Supplement to the Computational Text from A to Z Resource Guide**
+_Organized by the Equitable Syllabus Project_
+
+This document organizes machine learning tools for working with text—grouped by access model, type, and intended usage. It’s designed to help students, artists, and researchers choose appropriate tools for creative, ethical, and exploratory projects.
+
+---
+
+## 🔓 Open Weight / Self-Hosted Models
+
+These models can be downloaded and run locally, offering more transparency and experimentation potential.
+
+| Tool/Model                                               | Description                                                                                   | Tags                                       |
+| -------------------------------------------------------- | --------------------------------------------------------------------------------------------- | ------------------------------------------ |
+| **[Whisper](https://github.com/openai/whisper)**         | Open-source speech-to-text model by OpenAI. Great for transcription and voice-based projects. | `TTS/voice`, `open`, `offline`, `creative` |
+| **[LLaMA](https://ai.meta.com/llama/)**                  | Meta's open-weight large language model family. Popular for research and fine-tuning.         | `open`, `research`, `local`, `LLM`         |
+| **[GPT-Neo/GPT-J](https://huggingface.co/EleutherAI)**   | Open LLMs from EleutherAI. Can be run locally or hosted via HuggingFace.                      | `LLM`, `open`, `education`, `poetry`       |
+| **[StableLM](https://github.com/Stability-AI/StableLM)** | Stability AI's open-source LLM series for local generation tasks.                             | `creative`, `open`, `local`, `LLM`         |
+
+---
+
+## 🌐 API-Only / Cloud-Based Models
+
+Require API keys or subscriptions to use—less transparent, but powerful and production-ready.
+
+| Tool/Platform                                                     | Description                                                              | Tags                                             |
+| ----------------------------------------------------------------- | ------------------------------------------------------------------------ | ------------------------------------------------ |
+| **[OpenAI GPT-4 / ChatGPT](https://platform.openai.com/)**        | Leading closed-source LLM with natural dialogue and multi-modal support. | `closed`, `LLM`, `creative`, `education`         |
+| **[Google Gemini](https://deepmind.google/technologies/gemini/)** | Multimodal models for voice, vision, and language (via Bard or API).     | `closed`, `TTS/voice`, `AI writing`, `education` |
+| **[RunwayML](https://runwayml.com/)**                             | GUI-based creative ML platform—GPT, TTS, image gen, video tools.         | `closed`, `GUI`, `TTS`, `visual`, `creative`     |
```

Comment:
I don't believe Runway includes a language generation model. Also, for TTS, I would include [Elevenlabs](https://elevenlabs.io/). I've been experimenting with their API this summer and I plan on covering it in class.